src/styles/_header.scss

```diff
@@ -37,5 +37,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: color 300ms;
```

Comment:
```suggestion
    transition: color 0.3s;
```

Reply:
It's better to use one style for setting time

Reply:
changed